css/main.scss

```diff
@@ -196,7 +196,6 @@ footer .footer-block {
   position: relative;
   border-top: 75px solid transparent;
   margin-top: -75px!important;
-  pointer-events: none;
```

Comment:
Sorry about this one; the element doesn't overlap with anything clickable so we don't need this to prevent it being the invisible target of clicks.